scrapers_next/ca/people.py

```diff
@@ -56,17 +63,19 @@ def process_item(self, item):
 
             p.capitol_office.address = f"{cap_off_addr_part}" ", Sacramento, CA 95814"
 
-            po_box_and_phone = XPath("./td[4]/text()").match(item)[1].strip()
+            po_box_and_phone = header[5].text_content().strip()
             po_box, cap_phone = self.po_box_regex.search(po_box_and_phone).groups()
 
             p.capitol_office.voice = cap_phone
             p.extras["P.O. Box"] = po_box
 
             if "District Office" in h3_text:
-                dist_offs = header.getnext().text_content().strip()
                 dist_list = []
-                if len(dist_offs):
-                    dist_list = [x for x in dist_offs.split(" edit") if len(x)]
+                i = 6
+                while i < len(header):
+                    dist_list.append(header[i].text_content().strip())
+                    i = i + 1
+
```

Comment:
Interesting approach! This adds a few too many offices though since the index starts at a paragraph too high to match. I've gotten this to work if I set it to 
`i = 8
while i < (len(header)-1)`